kinode/packages/homepage/ui/src/store/persistentStore.ts

```diff
@@ -34,6 +34,9 @@ const usePersistentStore = create<PersistentStore>()(
       setFavoriteApps: (favoriteApps: PersistentStore['favoriteApps']) => set({ favoriteApps }),
       toggleWidgetVisibility: (package_name: string) => {
         const { widgetSettings } = get()
+        if (!window.confirm(`Really hide this widget?`)) {
```

Comment:
i really dislike this alert box. IMO it needs to either be an in-page confirmation or have no confirmation at all. i'm not sure why this action needs to be confirmed, given how straightforward it is to re-toggle the widget on.

another option would be to completely remove the "hide widget" button from the home screen and only allow that operation in the settings modal. that would be my preference.